data/card_catalogue/card-manipulation.json

```diff
@@ -4,77 +4,101 @@
     "upgrade": "Brainstorm",
     "tier": "blue",
     "details": {
-      "rules":"Cast two random Blue Cards.",
+      "rules": {
+        "active":"Draw 2 blue cards. Cast both.",
+        "passive": ""
+      },
       "flavour": "Live by the luck of the draw.",
-      "image": "assets/core/card/illustration/Bean Blast.png"
+      "image": "assets/core/card/illustration/Wing It.png"
     }
   },
   {
     "title": "Brainstorm",
     "upgrade": "Spark of Inspiration",
     "tier": "green",
     "details": {
-      "rules":"Cast two random Green Cards.",
+      "rules": {
+        "active":"Draw 2 green cards. Cast both.",
+        "passive": ""
+      },
       "flavour": "You never know what he'll pull out of thin air.",
-      "image": "assets/core/card/illustration/Bean Blast.png"
+      "image": "assets/core/card/illustration/Brainstorm.png"
     }
   },
   {
     "title": "Spark of Inspiration",
     "upgrade": "Randomize",
     "tier": "red",
     "details": {
-      "rules":"Cast two random Red Cards.",
-      "flavour": "I've got an idea",
-      "image": "assets/core/card/illustration/Bean Blast.png"
+      "rules": {
+        "active":"Draw 2 red cards. Cast both.",
+        "passive": ""
+      },
+      "flavour": "I've got an idea...",
+      "image": "assets/core/card/illustration/Spark of Inspiration.png"
     }
   },
   {
     "title": "Randomize",
     "upgrade": "",
     "tier": "gold",
     "details": {
-      "rules": "Cast two random Gold Cards.",
+      "rules": {
+        "active": "Draw 2 gold cards. Cast both.",
+        "passive": ""
+      },
       "flavour": "I don't know what I want, but I do want a lot of it.",
-      "image": "assets/core/card/illustration/Bean Blast.png"
+      "image": "assets/core/card/illustration/Randomize.png"
     }
   },
   {
     "title": "Grobble",
     "upgrade": "Slipshod Sorcery",
     "tier": "green",
     "details": {
-      "rules": "Draw 6 random Grey Cards.",
+      "rules": {
+        "active": "Draw 6 grey cards.",
```

Comment:
draw implies that it's from the face-down shuffled tier deck so we don't need to mention it